modules/ui.py

```diff
@@ -651,15 +638,6 @@ def open_folder(f):
                     with gr.Group():
                         html_info = gr.HTML()
                         generation_info = gr.Textbox(visible=False)
-                        if tabname == 'txt2img' or tabname == 'img2img':
-                            generation_info_button = gr.Button(visible=False, elem_id=f"{tabname}_generation_info_button")
-                            generation_info_button.click(
-                                fn=update_generation_info,
-                                _js="(x, y) => [x, y, selected_gallery_index()]",
-                                inputs=[generation_info, html_info],
-                                outputs=[html_info],
-                                preprocess=False
-                            )

```

Comment:
You may be undoing unrelated changes made in PR 4573.

Reply:
Oh no, I uploaded the files from an older version of WEBUI, I'll check for the changes